sysbuild/Kconfig.sysbuild

```diff
@@ -22,6 +22,7 @@ menuconfig PARTITION_MANAGER
 
 config PM_MCUBOOT_PAD
 	hex "Memory reserved for MCUBOOT_PAD"
+	default 0x800 if $(BOARD) = "nrf54l15pdk_nrf54l15_cpuapp"
```

Comment:
Acceptable as a quickfix, but we may think about better solution (this would not work for other nrf54l15 based boards)

Reply:
this needs hwmv2 to resolve